deploy/kubernetes-1.13/pmem-csi-direct.yaml

```diff
@@ -173,6 +173,7 @@ spec:
         - --v=3
         - --csi-address=/csi/csi-controller.sock
         - --feature-gates=Topology=true
+        - --strict-topology=true
```

Comment:
Even if we were to use "canary" images for kubernetes 1.14, we would still need images for 1.13.

Reply:
> > Canary images already having the change we need.
> 
> Yes, but we shouldn't switch to canary images in our "devel" branch, should we? Or are you suggesting that we do that?

With the comments you made on #221, I felt like you suggested to move to canary till  release-1.2 is ready.

Reply:
> Even if we were to use "canary" images for kubernetes 1.14, we would still need images for 1.13.

You mean csi-provisioner:master dropped support for 1.13? I didn't try myself on 1.13, as no cluster available.

Reply:
> > Even if we were to use "canary" images for kubernetes 1.14, we would still need images for 1.13.
> 
> You mean csi-provisioner:master dropped support for 1.13? I didn't try myself on 1.13, as no cluster available.

Ok, Now I got it. Both Kuberntes v1.13, and v1.14 using different provisioner releases. Yes, in that case, latest canary image might not work for 1.13.

Reply:
> With the comments you made on #221, I felt like you suggested to move to canary till release-1.2 is ready.

No, "tentatively change our deployment" wasn't meant as "change it on devel". I was thinking of a WIP branch.

> Both Kuberntes v1.13, and v1.14 using different provisioner releases. Yes, in that case, latest canary image might not work for 1.13.

I expects the 1.14 API for CSINodeInfo. It is useful to test whether it works on 1.13 despite that (would be great if it did, because then we don't need to backport into 1.0.x), but I am not optimistic.

> as no cluster available

You can use `TEST_CLEAR_LINUX_VERSION=<something old> CLUSTER=clear-govm-1-13 make start` to bring up one; I just don't have a sufficiently old version number at hand.